packages/dd-trace/test/telemetry/logs.spec.js

```diff
@@ -0,0 +1,283 @@
+const { expect } = require('chai')
+const proxyquire = require('proxyquire')
+
+describe('telemetry logs', () => {
+  const errorChannel = {
+    get hasSubscribers () {
+      return this.subscribe.callCount > 0
+    },
+    subscribe: sinon.stub(),
+    unsubscribe: sinon.stub()
+  }
+  const warnChannel = {
+    get hasSubscribers () {
+      return this.subscribe.callCount > 0
+    },
+    subscribe: sinon.stub(),
+    unsubscribe: sinon.stub()
+  }
+  const infoChannel = {
+    get hasSubscribers () {
+      return this.subscribe.callCount > 0
+    },
+    subscribe: sinon.stub(),
+    unsubscribe: sinon.stub()
+  }
+  const debugChannel = {
+    get hasSubscribers () {
+      return this.subscribe.callCount > 0
+    },
+    subscribe: sinon.stub(),
+    unsubscribe: sinon.stub()
+  }
```

Comment:
Instead of initialize these objects here and execute `reset` of all of them in beforeEach, can't we just init them in beforeEach method?